.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -9,3 +9,6 @@ contact_links:
   - name: 💬 Join us on Discord
     url: https://discord.gg/26P4xW4
     about: For realtime communication with the team and collaborators
+  - name: 📄 Blank Issue
+    url: https://github.com/iree-org/iree/issues/new
+    about: If you know what you're doing (especially collaborators)
```

Comment:
(reminder to remove one or the other before merge)

Reply:
Done (I appreciate these reminders, btw :slightly_smiling_face:)